onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -274,12 +274,6 @@
         "^test_string_concat_empty_string",
         "^test_string_concat_utf8",
         "^test_string_concat_zero_dimensional",
-        "^test_string_split_basic",
```

Comment:
Can we also check if those tests are commented out on C# side?

Reply:
Not quite sure how to check this.

Reply:
Could you point me towards how to check this? I'm not set up for C# development so it would be best if this could be validated by the CI.

Reply:
Could you point out how to check this?

Reply:
Disregard, we will do it another PR.